home/ivp-planning/advanced-tools/3d-import.md

```diff
@@ -0,0 +1,21 @@
+# 3D-Import
+
+To import a 3D file from your personal computer into iVP Planning, simply click on the "File" dropdown menu on the upper left side of the client, and select "Import".  
+
+![](../../../.gitbook/assets/3d-import-file.png)
+
+
+{% hint style="info" %}  
+For the best results, we recommend files in the [glTF](https://en.wikipedia.org/wiki/GlTF) format, which have the **.gltf** or **.glb** extension.  
+{% endhint %}
+
+
+Once you have selected a file, you get the option to define the files Unit of measurement (mm,, cm, dm, m, ft), and the direction of the Up Axis of the object.
+
+Please note that the up axis of the imported object/file must match with the Up Axis you have chosen when exporting the object/file.
+
+
+
+You will be able to change these options later via the properties panel, when highlighting the imported object/file.
```

Comment:
Could you make a single paragraph out of this section? The line breaks aren't really necessary